### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by omergulen**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-
+<head>
+  <link rel="shortcut icon" type="image/png" href="https://github.com/PortoTestersMeetup/portotestersmeetup.github.io/blob/master/images/icon/favicon.ico"/>
```
Comment (anthonydillon) [inline index.md]: Id recommend a relative link. For example, /images/icon/favicon.ico

**Comment by omergulen** [comment]
@anthonydillon done!

**Comment by FabioCB** [comment]
@omergulen Have you checked if it is working? by following this https://github.com/PortoTestersMeetup/portotestersmeetup.github.io/blob/master/CONTRIBUTING.md#how-to-use-your-github-repository-to-test

**Comment by omergulen** [comment]
yes

**Comment by FabioCB** [comment]
@omergulen thank you for your contribution. We're merging another PR that solves this issue and closing yours. Feel free to contribute on other [issues](https://github.com/PortoTestersMeetup/portotestersmeetup.github.io/issues)

**Pull request closed without merging**